src/Prelude/Bool/Properties.agda

```diff
@@ -5,15 +5,51 @@ open import Prelude.Equality
 open import Prelude.Bool
 open import Prelude.Decidable
 open import Prelude.Empty
+open import Prelude.Variables
+
+de-morg-neg-conj : (a : Bool) → (b : Bool) → not (a && b) ≡ (not a || not b)
+de-morg-neg-conj true true = refl
+de-morg-neg-conj false true = refl
+de-morg-neg-conj true false  = refl
+de-morg-neg-conj false false  = refl
+
+de-morg-neg-disj : (a : Bool) → (b : Bool) → not (a || b) ≡ (not a && not b)
+de-morg-neg-disj true true = refl
+de-morg-neg-disj false true = refl
+de-morg-neg-disj true false  = refl
+de-morg-neg-disj false false  = refl
+
+true≢false : _≢_ {A = Bool} true false
+true≢false ()
+
+false≢true : _≢_ {A = Bool} false true
```

Comment:
I defined them because of a special case in my 'Ord' extensions.
E.g https://gitlab.com/tmore/agda-prelude-extras/-/blob/master/src/Prelude/Extras/Ord.agda#L88
See the first case for the compare cases. (`a<?b≡false : true ≡ false`)
``` agda
  ≮?⇒≮ : {a b : A} → (a <? b) ≡ false → a ≮ b
  ≮?⇒≮ {a = a} {b = b} a<?b≡false
    with compare a b
  ...| less a<b = λ x → true≢false a<?b≡false
  ...| equal a≡b rewrite a≡b = λ a<b → ⊥-elim (less-antirefl a<b)
  ...| greater a>b = λ a<b → ⊥-elim (less-antisym a>b a<b)
```
Agda fails to solve the constraints for:
`(λ ()) a<?b≡false`

Reply:
I would write that as `case a<?b≡false of λ ()`.

Reply:
Fair enough, I will remove these